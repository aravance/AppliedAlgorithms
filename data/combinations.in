0 0
1 0
1 0
2 0
2 1
2 2
3 0
3 1
3 2
3 3
4 0
4 1
4 2
4 3
4 4
5 0
5 1
5 2
5 3
5 4
5 5
6 3
9 4
9 5
10 5
49 24
49 25
50 25
66 31
66 32
66 33
66 34
66 35
-1 -1

