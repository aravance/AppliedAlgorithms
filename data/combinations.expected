C(0,0) = 1
C(1,0) = 1
C(1,0) = 1
C(2,0) = 1
C(2,1) = 2
C(2,2) = 1
C(3,0) = 1
C(3,1) = 3
C(3,2) = 3
C(3,3) = 1
C(4,0) = 1
C(4,1) = 4
C(4,2) = 6
C(4,3) = 4
C(4,4) = 1
C(5,0) = 1
C(5,1) = 5
C(5,2) = 10
C(5,3) = 10
C(5,4) = 5
C(5,5) = 1
C(6,3) = 20
C(9,4) = 126
C(9,5) = 126
C(10,5) = 252
C(49,24) = 63205303218876
C(49,25) = 63205303218876
C(50,25) = 126410606437752
C(66,31) = 6406484391866534976
C(66,32) = 7007092303604022630
C(66,33) = 7219428434016265740
C(66,34) = 7007092303604022630
C(66,35) = 6406484391866534976
